### PR TITLE
Dev2

### PR DESCRIPTION
**Pull request by noah-puckett**

### CODE REVIEW

File: classes.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+//class that is modeling data
```
Comment (MackenzieBerliner-Glasser) [inline classes.js]: Is there a reason that this file is in your solo build?

File: old-index.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```
Comment (MackenzieBerliner-Glasser) [inline old-index.html]: if this is an old file that is no longer being used I would not include it in production.

File: src/components/App.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+//import other components to render
+import Header from '../components/Header.js';
+import Component from './Component.js';
+import AddItem from '../components/AddItem.js';
+//todoData is our data array, not a component
+import todoData from '../../data/todo-data.js';
+import TodoList from './TodoList.js';
+
+//THIS FILE RENDERS OUR WEBSITE DYNAMICALLY
```
Comment (MackenzieBerliner-Glasser) [inline src/components/App.js]: The concepts are very new, I understand the need to take notes. This does make the PR crowded and hard to read. In the future we will be a lot more critical of comments as well as commenting out any code.

File: src/old-index.js [hunk 4]
```diff
@@ -1,3 +1,6 @@
+//do I need this here??? is this only for the new index? TODO: what the heck
```
Comment (MackenzieBerliner-Glasser) [inline src/old-index.js]: Again, old files should not be added to production.

Comment (noah-puckett) [reply src/old-index.js]: Fixed!

File: src/components/App.js (hunk 3 above)
Comment (noah-puckett) [reply src/components/App.js]: Thanks for letting me know, I'll be better about cleaning up code before pushing to origin :)

File: old-index.html (hunk 2 above)
Comment (noah-puckett) [reply old-index.html]: Gotcha! Edited out.

File: classes.js (hunk 1 above)
Comment (noah-puckett) [reply classes.js]: I'm not going to lie, I only barely remember writing that at all? Anyways, deleted! Thanks for catching that!

**Pull request merged**